client/stylesheets/_responsive.scss

```diff
@@ -471,6 +481,10 @@
     .homeCardDetails {
         margin-top: 0;
     }
+
+    .profileCardLi {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardLi` should be written in lowercase with hyphens